### PR TITLE
Add test failure page, fix some null issues, and date formatting!

### PR DESCRIPTION
**Pull request by Pilchie**

### CODE REVIEW

File: Shared/IssueEntity.cs [hunk 1]
```diff
@@ -1,35 +1,42 @@
...
 {
+    [DebuggerDisplay("{GetDebuggerDisplay(),nq}")]
     public class IssueEntity : TableEntity
     {
         public IssueEntity()
         { }
 
         public IssueEntity(Issue issue)
-            : base(DateTime.UtcNow.Date.ToString("yyyy-mm-dd"), issue.Number.ToString("D"))
+            : base(DateTime.UtcNow.Date.ToString("yyyy-MM-dd"), issue.Number.ToString("D"))
```
Comment (Pilchie) [inline Shared/IssueEntity.cs:18]: I hate my life.

**Review by Pilchie — COMMENTED** [review]
Not sure why TestFailures files are binary :(

**Comment by Pilchie** [comment]
Fixed line endings.  Files were in utf-16 :(
